yaml/earth.yaml

```diff
@@ -0,0 +1,79 @@
+# synthetic columns are values available to filters and output, but which are
+# generated in the SQL function and not passed in from the tables. in this case,
+# we generate the volume from the building height and way_area, where available.
+synthetic_columns:
+  - test
```

Comment:
If I remove the `synthetic_columns` block here, I get `way_area` related errors :/